database/ticket_test.go

```diff
@@ -83,6 +86,32 @@ func testInsertNewTicket(t *testing.T) {
 	}
 }
 
+func testHasFeeAddress(t *testing.T) {
+	ticket := exampleTicket()
+	has, err := db.HasFeeAddress(ticket.FeeAddress)
+	if err != nil {
+		t.Fatalf("error retrieving has fee address: %v", err)
+	}
+	if has {
+		t.Fatal("unexpected has for fee address")
+	}
+
+	// Insert a ticket into the database.
+	err = db.InsertNewTicket(ticket)
+	if err != nil {
+		t.Fatalf("error storing ticket in database: %v", err)
+	}
+
+	has, err = db.HasFeeAddress(ticket.FeeAddress)
+	if err != nil {
+		t.Fatalf("error retrieving has fee address: %v", err)
+	}
+	if !has {
+		t.Fatal("unexpected has for fee address")
+	}
+
+}
+
```

Comment:
I think the error messages could be more descriptive here, without looking at the code `has` in the error messages has very little context. For example,`unexpected has for fee address` could be `expected db to have ticket with fee address <addr>`.

Reply:
ok